models/intel/action-recognition-0001/description/action-recognition-0001.md

```diff
@@ -0,0 +1,73 @@
+# Composite model name: action-recognition-0001
```

Comment:
This should be consistent with the non-composite models. I would suggest something like:

```markdown
# action-recognition-0001 (composite)
```

Reply:
Done for all models